html/Good_practices_in_Web_Dev/good-practices-in-web-dev.md

```diff
@@ -0,0 +1,248 @@
+# Good practices in Web-Dev
+
+To make sure quality user experience, your website must be displayed correctly across all the devices. To achieve this, we need to write scalable code and give the same output for all the devices.
+
+## what will it change?
+Following this will, of course, enhance the user experience once they visit your web platform. It will improve the readability of your code with the help of clear indentations, and depending upon the performance, it will also enhance the SEO rankings, pushing your platform to the top in search engines. It'll boost up the accessibility on the screen reader and other identical technologies.
+
+
+## Tell me Some examples
```

Comment:
Check above comment, first set theme then expand around that and then provide example

- SEO
    - Some info on how SEO ranking works and what you can do about it. Learn about meta tags. Vilva Athiban has a nice tutorial series around it. https://www.youtube.com/watch?v=I5PtXNS_EoE 
    - code example
- Accessibility
    - here you can talk about semantic HTML and which tags to use in which cases. Things like don't use a tag with href="#" to create a button, this hurts accessibility
    - use case of when to use div
    - other tags followed by code snippet
 ... continue in similar way